railties/test/generators/api_app_generator_test.rb

```diff
@@ -141,6 +141,7 @@ def default_files
         app/views/layouts
         app/views/layouts/mailer.html.erb
         app/views/layouts/mailer.text.erb
+        bin/dev
```

Comment:
Now that there is a [script/](https://github.com/rails/rails/pull/52335) directory generator, would it be more appropriate for this file to be defined as `script/dev`?

The initial version of `bin/dev` is Rails-generated, but has the potential to evolve as an application's needs (and therefore demands on the code in the script) evolve. 

The same isn't true for most of its peer files in `bin/`, since they're either Bundler- or Rails-generated binstubs that likely will not change.

Reply:
No, bin/ is for binstubs like this that start a command like server. script/* is for app specific scripts like recurring crons or one-off migrations or the like. We need to be clear about this in the documentation.

Reply:
Does `bin/setup` belong in `script/`?